src/coreclr/jit/targetx86.h

```diff
@@ -54,8 +54,10 @@
   #define FEATURE_EH               1       // To aid platform bring-up, eliminate exceptional EH clauses (catch, filter,
                                            // filter-handler, fault) and directly execute 'finally' clauses.
 
-  #define FEATURE_EH_CALLFINALLY_THUNKS 0  // Generate call-to-finally code in "thunks" in the enclosing EH region,
-                                           // protected by "cloned finally" clauses.
+#if !defined(UNIX_X86_ABI)
+  #define FEATURE_EH_X86_FRAMES    1       // Enable support for SEH regions
```

Comment:
This name is confusing to me. Isn't it basically `!FEATURE_EH_FUNCLETS`? And how does "Enable support for SEH regions" describe it?

Could it be `FEATURE_EH_WINDOWS_X86`?

I guess the point is there is some code that is statically known to never be needed unless it's win-x86. Since funclets or not will be dynamic, we need to compile it in if it might be needed. But it will never get called except on win-x86 when `UsesFunclets()` is true.

Reply:
I can rename it. Yep, it's basically a shorthand for `TARGET_X86 && !UNIX_X86_ABI` to enable the optional SEH code path.